ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/Constants.java

```diff
@@ -46,11 +46,10 @@
   public static long GENESIS_EPOCH = 0;
   public static long GENESIS_FORK_VERSION = 0;
   public static UnsignedLong FAR_FUTURE_EPOCH = UnsignedLong.MAX_VALUE;
-  public static Bytes32 ZERO_HASH = Bytes32.ZERO; // TODO confirm if equals to b'\x00' * 32
```

Comment:
Can you add this back? I know it's removed in the spec, but we use it when initializing fields of Bytes32 values when they are empty.